doc/lsp.txt

```diff
@@ -351,6 +351,23 @@ usePopupInCodeAction    |Boolean| option.  When using the |:LspCodeAction|
 			command to display the code action for the current
 			line, use a popup menu instead of echoing.
 			By default this is set to false.
+diagSignErrorText       |String| option. Change diag sign text for errors
```

Comment:
The list of options in the help is alphabetically sorted.  Can you move the below text to the right place above?

Reply:
fix added in https://github.com/yegappan/lsp/pull/190/commits/680004fed5a5eca138662e3d7ba5b4a514a18ad3